client/ayon_silhouette/api/plugin.py

```diff
@@ -60,11 +61,32 @@ def create(self, product_name, instance_data, pre_create_data):
         if not session:
             return
 
-        # Create new node and place it in the scene
-        instance_node = fx.Node(self.node_type)
-        instance_node.label = session.uniqueLabel(product_name)
-        session.addNode(instance_node)
-        lib.set_new_node_position(instance_node)
+        instance_node = None
+        if pre_create_data.get("use_selection"):
```

Comment:
It seems like this change had not fixed the situation with unconnected node creation.

Reply:
> It seems like this change had not fixed the situation with unconnected node creation.

Could you describe what you mean?

1. When clicking Create without any **valid** node (of the correct type) selected it will still generate a new node.
2. When clicking Create with a valid node type selected it will now use that node to publish from - so that you could e.g. publish an existing roto node instead of needing to generate a new node.

But reading this now.. it sounds like what you expected was that the newly created node would just _connect_ (as input) to the user's selection?

Reply:
> But reading this now.. it sounds like what you expected was that the newly created node would just connect (as input) to the user's selection?

Corret. Not sure what is correct approach but for example if I want to create render product I am selecting Read node or Roto node and neither of those is linking the newly created render node.